scene/2d/tile_map.cpp

```diff
@@ -35,6 +35,31 @@
 #include "os/os.h"
 #include "servers/physics_2d_server.h"
 
+Rect2 TileMap::_edit_get_rect() const {
+
+	Rect2 map_size = _compute_used_size();
```

Comment:
Wouldn't it be better to use `get_used_rect()` and have the cached size instead of recomputing it all the time?

Reply:
_edit_get_rect needs to be const, and cannot modify the tilemap members. It could read the cache status, and only call if it's dirty, but then it wouldn't be able to write the new cache.